coordinator/src/main/scala/filodb.coordinator/queryplanner/PlannerMaterializer.scala

```diff
@@ -183,7 +183,8 @@ trait  PlannerMaterializer {
     val aggregate = Aggregate(AggregationOperator.Sum, lp, Nil, Seq("job"))
     // Add sum to aggregate all child responses
     // If all children have NaN value, sum will yield NaN and AbsentFunctionMapper will yield 1
-    val aggregatePlanResult = PlanResult(Seq(addAggregator(aggregate, qContext, vectors, Seq.empty)))
+    val aggregatePlanResult = PlanResult(Seq(addAggregator(aggregate, qContext.copy(plannerParams =
+      qContext.plannerParams.copy(skipAggregatePresent = true)), vectors, Seq.empty))) // No need for present for sum
```

Comment:
Now that we have this, should the canHandleEmptyResult be removed from RangeVectorTransformer ?

Reply:
We still need `canHandleEmptyResult` for AbsentFunctionMapper as the top level exec plan (ReduceAggrgateExec) will have AbsentFunctionMapper and it will not get executed for empty schema if canHandleEmptyResult is false